Jenkinsfile

```diff
@@ -16,8 +16,8 @@ pipeline {
     productionHost = credentials('PRODUCTIONHOST')
   }
   parameters {
-    choice(choices: ['validate', 'create'], description: 'Behaviour at connection time (initialize/validate schema)',
-     name: 'dbBehaviour')
+    choice(choices: ['validate', 'update','create'], description: 'Behaviour at connection time for staging only \
+           (initialize/update/validate schema)', name: 'dbBehaviour')
```

Comment:
`ddlBehaviour` instead of `dbBehaviour` to make it more specific and match the name of the actual argument?